spec/components/search_result/document_component_spec.rb

```diff
@@ -3,13 +3,25 @@
 require "rails_helper"
 
 RSpec.describe SearchResult::DocumentComponent, type: :component do
-  pending "add some examples to (or delete) #{__FILE__}"
-
-  # it "renders something useful" do
-  #   expect(
-  #     render_inline(described_class.new(attr: "value")) { "Hello, components!" }.css("p").to_html
-  #   ).to include(
-  #     "Hello, components!"
-  #   )
-  # end
+  let(:document) do
+    SolrDocument.new(
+      id: 'x',
+      eds_title:
+    )
+  end
+
+  let(:component) { described_class.new(document:) }
```

Comment:
I believe `document:` should be passed a presenter. (confusing!) https://github.com/projectblacklight/blacklight/blob/main/app/components/blacklight/document_component.rb#L80

Reply:
Very. Explains why it kept breaking on CI. Not sure why it was passing locally 🤷 . Anyway that fixed the failing test. Thanks.